code/game/objects/items/melee/transforming.dm

```diff
@@ -61,8 +61,13 @@
 			attack_verb = attack_verb_off
 		icon_state = initial(icon_state)
 		w_class = initial(w_class)
-	GET_COMPONENT_FROM(butchering, /datum/component/butchering, src)
-	butchering.butchering_enabled = active
+	if(is_sharp())
```

Comment:
not doing a single get component/load component because don't need to add a component if it has none if it's not even sharp. if it has one and it's no longer sharp, disable it.